### ISSUE TITLE
Files that are named like Rust keywords don't work

### ISSUE THREAD

**Issue by BinderDavid** [binary-banter/test-each-file#7]
I use the macro to test a compiler, and the testsuite contains files that are named like Rust keywords. For example, `match.xxx`, `let.xxx` etc. This leads to an error in the macro with the following error message:

```console
error: expected identifier, found keyword `match`
  --> lang/fun/tests/integration_test.rs:43:1
   |
43 | test_each_file!(in "./testsuite/success" as typecheck_success => typecheck_test);
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected identifier, found keyword
   |
   = note: this error originates in the macro `test_each_file` (in Nightly builds, run with -Z macro-backtrace for more info)
help: escape `match` to use it as an identifier
   |
43 | r#test_each_file!(in "./testsuite/success" as typecheck_success => typecheck_test);
   | ++

```

**Comment by JonathanBrouwer** [comment]
Will take a look at this later tonight, should be easy to fix, thanks for reporting!

**Comment by JonathanBrouwer** [comment]
Should be fixed in  test_each_file v0.3.4, which is now on crates.io. Thanks for reporting!

**Comment by BinderDavid** [comment]
Thank you very much for your super speedy response and fix :)
